src/style.scss

```diff
@@ -94,6 +94,12 @@ body {
   color: white;
 }
 
+.sidebar-wrapper {
```

Comment:
this is all duplicated from the sidebar styles, which are in the `src/styles/sidebar.scss`, see below